src/net.cpp

```diff
@@ -1053,6 +1053,8 @@ bool CConnman::AddConnection(const std::string& address, ConnectionType conn_typ
     case ConnectionType::BLOCK_RELAY:
         max_connections = m_max_outbound_block_relay;
         break;
+    case ConnectionType::FULLRBF:
+        max_connections = m_max_outbound_fullrbf_relay;
```

Comment:
There is a `break` missing, so this would fallthrough.

Reply:
Correct.